src/main/app/form/validation.ts

```diff
@@ -88,3 +88,13 @@ export const isInvalidHelpWithFeesRef: Validator = value => {
     }
   }
 };
+
+export const isPhoneNoValid: Validator = value => {
+  if (!value) {
```

Comment:
is this necessary? what case does it catch?

Reply:
👍 I've moved this falsy check into the RegEx in 6b31e94.